src/LayoutManager/LayoutManager.ts

```diff
@@ -106,16 +72,56 @@ export class LayoutManager {
     this._prevLayoutStrategy = strictContext.strategy;
   }
 
+  /**
+   * Attach event handlers to actions we know are layout invalidating when
+   * performed on child objects. returns the disposers for later unsubscribing
+   * @param {FabricObject} childObject
+   * @param {RegistrationContext & Partial<StrictLayoutContext>} context
+   * @returns {VoidFunction[]} disposers remove the handlers
+   */
   protected attachHandlers(
     childObject: FabricObject,
     context: RegistrationContext & Partial<StrictLayoutContext>
```

Comment:
This type doesn't make sense
The point of StrictLayoutContext is that it is not partial IIRC

Reply:
There is a story for this i need to check